structuredefinitions/Extension-UKCore-CodingSCTDesc.xml

```diff
@@ -0,0 +1,93 @@
+  <status value="active" />
+  <date value="2022-08-26" />
+  <publisher value="HL7 UK" />
+  <contact>
+    <name value="HL7 UK" />
+    <telecom>
+      <system value="email" />
+      <value value="secretariat@hl7.org.uk" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <contact>
+    <name value="NHS Digital" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="2" />
+    </telecom>
+  </contact>
+  <description value="The description display term for a SNOMED CT concept.  The description display term associated with the ID within the sctdescid extension is important to enable the representation of the exact term that was entered by the clinician that recorded the data, or the exact term that a legacy code e.g. a Read v2 code has been translated to using an assured mapping table." />
+  <purpose value="This extension extends the sctdescid extension to support the exchange of the selected description display term for a SNOMED CT concept, which is currently not supported by the FHIR standard." />
+  <copyright value="Copyright © 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="rim" />
+    <uri value="http://hl7.org/v3" />
+    <name value="RIM Mapping" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="coding-sctdescid.value" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+	<differential>
+	 <element id="Extension">
+	<path value="Extension"/>
+	<short value="SNOMED CT Description ID"/>
+	<definition value="To record the SNOMED CT Description Display term actually selected"/>
+	<max value="1"/>
+	</element>
+	 <element id="Extension.extension">
+	<path value="Extension.extension"/>
+	 <slicing>
+	 <discriminator>
+	<type value="value"/>
+	<path value="url"/>
+	</discriminator>
+	<rules value="open"/>
+	</slicing>
+	<min value="1"/>
+	</element>
+	 <element id="Extension.extension:descriptionDisplay">
```

Comment:
I dont think you need to slice here, I think using the same structure as the international extension **coding-sctdescid** is probably what we want here, as the display will be used in the same way.  See here for the StructureDefinition of **coding-sctdescid** https://hl7.org/fhir/r4/extension-coding-sctdescid.json.html

Reply:
> ink using t

I htought that, but was using the old UKCore SCTDescid as a base. Now reworked to use the FHIR version structure